src/csharp/Microsoft.Spark.Worker/TaskRunner.cs

```diff
@@ -47,6 +49,11 @@ public void Run()
         {
             s_logger.LogInfo($"[{TaskId}] Starting with ReuseSocket[{_reuseSocket}].");
 
+            if (EnvironmentUtils.GetEnvironmentVariableAsBool("DEBUG_DOTNET_WORKER"))
```

Comment:
```suggestion
            if (EnvironmentUtils.GetEnvironmentVariableAsBool("DOTNET_WORKER_DEBUG"))
```

Follow same pattern as `DOTNET_WORKER_SPARK_VERSION` and `DOTNET_WORKER_DIR`